plasma_framework/contracts/src/exits/payment/PaymentStandardExitable.sol

```diff
@@ -14,7 +18,11 @@ import "../../utils/Merkle.sol";
 import "../../transactions/PaymentTransactionModel.sol";
 import "../../transactions/outputs/PaymentOutputModel.sol";
 
-contract PaymentStandardExitable is OnlyWithValue {
+contract PaymentStandardExitable is
```

Comment:
Should this contract also be a `IExitProcessor`?
In `enqueueStandardExit()` it assigns itself to be the exitProcessor:
```
 ExitModel.Exit memory exitDataForQueue = ExitModel.Exit({
            exitProcessor: address(this),
```

Reply:
ah...good point. Would add that.